bin.js

```diff
@@ -594,7 +594,7 @@ async function runTool (args, Tool, version, uiOptions) {
       rimraf.sync(outputs.visualizer)
     }
   } else if (outputs.visualizer) {
-    console.log(`Generated HTML file is ${outputs.visualizer}`)
+    console.log(`Generated HTML file is file://${path.resolve(outputs.visualizer)}`)
```

Comment:
This is unlikely to do the right thing on all platforms. You'll want to use [url.pathToFileURL(path)](https://nodejs.org/dist/latest-v13.x/docs/api/url.html#url_url_pathtofileurl_path) to generate the file URL appropriately. The challenge is that just using `path.resolve()` is going to give you an operating system specific result that will yield invalid file URLs. `pathToFileURL()` normalizes those for you.

Reply:
Ohh I didn't know about that API!